src/hera/shared/_global_config.py

```diff
@@ -41,6 +42,11 @@ class _GlobalConfig:
     _token: Optional[Union[str, TokenGenerator, Callable[[], Optional[str]]]] = None
     """an optional authentication token used by Hera in communicating with the Argo server"""
 
+    _client_certs: Optional[
```

Comment:
What do people think of having a custom type (similar to custom generator) to capture all the variations? Or is there one already we can use? I know we're passing through to `requests.get` but I'm not super familiar with SSL/certs standards/patterns, just wondering if the pair is common enough.

Reply:
@elliotgunton  AFAIK `requests` always expect a path of  (cert,key) `str` pair. So perhaps a custom type might not be useful in this case? I could be wrong though